Makefile.in

```diff
@@ -228,6 +228,13 @@ config_files: ocaml-stage1-config.status
 	/bin/echo -n "( `grep \"^SHAREDLIB_CFLAGS=\" ocaml/Makefile.config \
 		| sed 's/^SHAREDLIB_CFLAGS=//'` )" > sharedlib_cflags.sexp
 
+## For targets not supported in backend, we can install the result of stage0
+.PHONY: install_stage0
+install_stage0: stage0
+	rsync --copy-links --chmod=u+rw,go+r -r $(stage0_prefix)/* $(prefix)
+        # CR gyorsh: when is it supposed to be installed for upstream build?
```

Comment:
I don't know the answer to this question -- presumably just install it at the same time as we do at the moment?

Reply:
I removed the CR in the latest commit.